specification/monitor/data-plane/Microsoft.Insights/preview/2023-03-01-preview/metricBatch.json

```diff
@@ -183,85 +197,106 @@
+                    "type": "string"
+                  },
+                  "info": {
+                    "description": "Additional information related to the error.",
+                    "type": "string"
+                  }
+                }
               }
+            },
+            "code": {
+              "description": "Error code",
+              "type": "string"
+            },
+            "message": {
+              "description": "Error message indicating why the operation failed.",
+              "type": "string"
             }
           }
-        },
-        "code": {
-          "description": "Error code",
-          "type": "string"
-        },
-        "message": {
-          "description": "Error message indicating why the operation failed.",
-          "type": "string"
         }
       }
     },
     "MetricResultsResponse": {
       "description": "The metrics result for a resource.",
-      "type": "array",
-      "items": {
-        "type": "object",
-        "properties": {
-          "timespan": {
-            "type": "string",
-            "description": "The timespan for which the data was retrieved. Its value consists of two datetimes concatenated, separated by '/'.  This may be adjusted in the future and returned back from what was originally requested."
-          },
-          "interval": {
-            "type": "string",
-            "format": "duration",
-            "description": "The interval (window size) for which the metric data was returned in.  This may be adjusted in the future and returned back from what was originally requested.  This is not present if a metadata request was made."
-          },
-          "namespace": {
-            "type": "string",
-            "description": "The namespace of the metrics been queried"
-          },
-          "resourceregion": {
-            "type": "string",
-            "description": "The region of the resource been queried for metrics."
-          },
-          "resourceid": {
-            "type": "string",
-            "description": "The resource that has been queried for metrics."
-          },
-          "value": {
-            "type": "array",
-            "items": {
-              "$ref": "#/definitions/Metric"
+      "type": "object",
+      "properties": {
+        "values": {
+          "description": "The collection of metric data responses per resource, per metric.",
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "timespan": {
+                "type": "string",
+                "description": "The timespan for which the data was retrieved. Its value consists of two datetimes concatenated, separated by '/'.  This may be adjusted in the future and returned back from what was originally requested."
+              },
+              "interval": {
+                "type": "string",
+                "format": "duration",
```

Comment:
This appears to be an OpenAPI 3.0 format. We support only OpenAPI 2.0 aka "swagger". Also, see https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#:~:text=DO%20use%20a%20fixed%20time%20interval%20to%20express%20durations. Ideally, this is something like `internalInSeconds` or whatever is most appropriate.